lib/ariadne/data_util.rb

```diff
@@ -1,11 +1,12 @@
 
 module DataUtil
-  def self.init_redis_cli(obj = nil)
-    @redis_cli = obj
+  def self.init_redis_cli(redis_obj: nil)
+    @redis_cli = redis_obj
   end
 
-  def self.get_data_from_redis(app_name)
-    keys = @redis_cli.keys "#{app_name}*"
+  def self.get_data_from_redis(id:, app_name:)
+    key  = id.nil? ? "#{app_name}*" : "#{app_name}:#{id}"
+    keys = @redis_cli.keys key
```

Comment:
I am not sure why I dont like the way this has been written

The caller of this method already knows if `id` is `nil` and hence can pass suitable value for `key`. Why do we need to parameters?